static/index.html

```diff
@@ -57,6 +57,31 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. You can also extend these with
             support for your own builds.</p>
+            <h2 id="overview"><a href="#overview">Overview</a></h2>
+            <ul>
+                <li>
+                    GrapheneOS does <em>not</em> support Google play services. Some apps may not work, or only have partial functionality.
+                </li>
+                <li>
+                    GrapheneOS does <em>not</em> support rooting. This goes against its design goals and will <b>never</b> be supported.
+                </li>
+                <li>
+                    If you are looking to buy a phone to run GrapheneOS, the Pixel 3(xl) and 3a(xl) are the best options.
+                </li>
+                <li>
+                    GrapheneOS currently has no plans to support the Pixel 4 or Pixel 4XL
+                </li>
+                <li>
+                    This OS is 'barebones' and does not come with many preinstalled apps ontop of AOSP.
```

Comment:
It's already stated that the OS is in an early stage of development and I don't want to state things like this as if it's part of the goals rather than a consequence of that.

Reply:
I agree with this sentiment, the way I phrased it made it seem like a "downside" of the project.